Resources/Textures/Objects/Weapons/Throwable/bola.rsi/meta.json

```diff
@@ -0,0 +1,14 @@
+{
+    "version": 1,
+    "license": "",
```

Comment:
As suggested in comments, head to the codebase github page readme you took the icon from. For paradise for example if you scroll right to the bottom it states all icons are creative commons 3.0 so you'd stick "CC-BY-SA-3.0" in the license box.